glossary.rst

```diff
@@ -0,0 +1,67 @@
+Glossary
+========
+
+.. glossary::
+
+    Application
+        Website or application tracked by PPMS.
+
+    App ID
+        PPMS :term:`application` identificator (previously **website ID**, **site ID** or **idSite**).
+
+    User
+        Visitor on tracked :term:`application`.
+
+    Analytics ID
+        ID assigned to :term:`user` by :term:`Analytics` for the duration of :term:`Analytics` session. It's stored in
+        browser cookie.
+
+    User ID
+        Permanent ID assigned to :term:`user` by :term:`application` (e.g. username). You can read more about it
+        `here <https://help.piwik.pro/tag-manager/userid/>`_.
+
+    Device ID
+        Device ID (device identification) is a distinctive number associated with a smartphone or similar handheld
+        device. Device IDs are separate from hardware serial numbers.
+
+    Identifier
+        Unique :term:`user` ID (e.g. :term:`analytics ID`, :term:`user ID`, :term:`device ID` or email).
+
+    Visit
+        Period of continuous :term:`user` activity on :term:`application`. Its default maximal length is 30 minutes.
+
+    Audience
+        Named set of :term:`attribute` conditions used to define a group of :term:`users<user>` matching them.
+
+    Attribute
+        Named value assigned to :term:`user` profile.
+
+    Attribute whitelist
+        List of :term:`user` :term:`attributes<attribute>` that are publicly available via Audience Manager get
+        :term:`attribute` API. To access these information's it's necessary to provide :term:`analytics ID`.
+
+    PII
+        Personally Identifiable Information.
+
+    Analytics attribute
+        :term:`Attribute` generated from value provided by :term:`Analytics` (e.g. browser and device data, location
+        data, etc.). You can read more about :term:`attribute` sources
+        `here <https://help.piwik.pro/audience-manager/data-sources/>`_.
+
+        .. note::
+            If :term:`custom attribute` will use same name - it will be represented as separate :term:`attribute`.
+
+    Custom attribute
+        :term:`Attribute` generated from value provided by source other than :term:`Analytics` (e.g.
+        :doc:`audience_manager/form-tracker`, :doc:`sdk/index`). You can read more about :term:`attribute` sources
+        `here <https://help.piwik.pro/audience-manager/data-sources/>`_.
+
+        .. warning::
+            Data using same name send from different custom sources will override value of same
```

Comment:
This sentence is weird, something like "only the most recent value is stored for each custom attribute" would probably be more clear.